iodata/formats/pdb.py

```diff
@@ -62,66 +60,116 @@ def load_one(lit: LineIterator) -> dict:
     # 61 - 66        Real(6.2)     tempFactor   Temperature  factor.
     # 77 - 78        LString(2)    element      Element symbol, right-justified.
     # 79 - 80        LString(2)    charge       Charge  on the atom.
+
+    # Get element symbol from position 77:78 in pdb format
+    words = line[76:78].split()
+    if not words:
+        # If not present, guess it from position 13:16 (atom name)
+        words = line[12:16].split()
+    # assign atomic number
+    symbol = words[0].title()
+    atnum = sym2num.get(symbol, sym2num.get(symbol[0], None))
+    # atom name, residue name, chain id, & residue sequence number
+    attype = line[12:16].strip()
+    restype = line[17:20].strip()
+    chainid = line[21]
+    resnum = int(line[22:26])
+    # add x, y, and z
+    atcoord = [
+        float(line[30:38]) * angstrom,
+        float(line[38:46]) * angstrom,
+        float(line[46:54]) * angstrom,
+    ]
+    # get occupancies & temperature factor
+    occupancy = float(line[54:60])
+    bfactor = float(line[60:66])
+    return atnum, attype, restype, chainid, resnum, atcoord, occupancy, bfactor
+
+
+def _parse_pdb_conect_line(line):
+    # Overview of CONECT records
+    # COLUMNS       DATA  TYPE      FIELD        DEFINITION
+    # -------------------------------------------------------------------------
+    #  1 -  6       Record name    "CONECT"
+    #  7 - 11       Integer        serial       Atom  serial number
+    # 12 - 16       Integer        serial       Serial number of bonded atom
+    # 17 - 21       Integer        serial       Serial number of bonded atom
+    # 22 - 26       Integer        serial       Serial number of bonded atom
+    # 27 - 31       Integer        serial       Serial number of bonded atom
+    iatom0 = int(line[7:12]) - 1
+    for ipos in 12, 17, 22, 27:
+        serial_str = line[ipos: ipos + 5].strip()
+        if serial_str != "":
+            iatom1 = int(serial_str) - 1
+            if iatom1 > iatom0:
+                yield iatom0, iatom1
+
+
+@document_load_one("PDB", ['atcoords', 'atnums', 'atffparams', 'extra'], ['title', 'bonds'])
+def load_one(lit: LineIterator) -> dict:
+    """Do not edit this docstring. It will be overwritten."""
+    title = "PDB file from IOData"
     atnums = []
     attypes = []
     restypes = []
     chainids = []
     resnums = []
-    coords = []
+    atcoords = []
     occupancies = []
     bfactors = []
+    bonds = []
     molecule_found = False
     end_reached = False
-    title = "PDB file from IOData"
     while True:
         try:
             line = next(lit)
         except StopIteration:
             break
-        # If the PDB file has a title replace it.
+        # If the PDB file has a title, replace the default.
         if line.startswith("TITLE") or line.startswith("COMPND"):
             title = line[10:].rstrip()
```

Comment:
If the PDB has a `TITLE` or `COMPND` section, they are usually split over multiple lines, and I think the current code only keeps the last line. So, shouldn't we use sth like: `title += line[10:].rstrip()`?

Reply:
That would indeed be good. We may still include the newline characters, such that the title can be written out over multiple lines as well.